src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
+
+namespace Arriba.Diagnostics.Tracing
+{
+    public class ArribaLogs
```

Comment:
I would extract a small interface from this with the Write* methods and make the controllers and other stuff depend on that instead

Reply:
Agree, I'll create a separate PR for this.